templates/vm/votp_vm_rt_isolated.xml.j2

```diff
@@ -61,7 +61,11 @@
             <target type="serial" port="0" />
         </console>
         <memballoon model="virtio">
+	    <stats period="5" />
         </memballoon>
+	<vsock model='virtio'>
+            <cid auto='yes' />
```

Comment:
Maybe we will need to force this ID to be able to specify which VSOCK does what from the guest point-of-view.

Reply:
the guest can only have one vsock anyway... as long as this "auto" approach works and does not create problem, I'm in favor of keeping it.
Just so you know I tested with several guests on the same host and it also works.